src/app/app.tsx

```diff
@@ -12,6 +12,7 @@ import { parse } from 'semver';
 import { Card } from './components/card';
 import { Navbar } from './components/navbar';
 import { useUrlParam } from './hooks/url-params';
+import Table from './components/table';
```

Comment:
None of the other components use a default export - I recognize that they are pretty prevalent in react ecosystem, but for now lets stick with named.